packages/frontend/src/pages/Login.tsx

```diff
@@ -232,7 +235,34 @@ const Login: FC<{ minimal?: boolean }> = ({ minimal = false }) => {
             <LoginContent />
         </Stack>
     ) : (
-        <Page title="Login" withCenteredContent withNavbar={false}>
+        <Page
+            title="Login"
+            withCenteredContent
+            header={
+                useNewLogin && (
+                    <Header
+                        height={NAVBAR_HEIGHT}
+                        styles={(theme) => ({
+                            root: {
+                                display: 'flex',
+                                justifyContent: 'center',
+                                backgroundColor: theme.colors.blue['6'],
+                                color: 'white',
+                            },
+                        })}
+                    >
+                        <Center mx="sm">
+                            <Text weight={500} ta="center">
+                                Our login page looks different! We’ve made
+                                changes to login to improve your experience.
+                                Please provide your email below or login with
+                                your SSO provider.
```

Comment:
they might not see their SSO provider until they type in their email (I think this is the main source of confusion). so we shouldn't say "login with your SSO provider"

Reply:
Done